charts/cluster-api-provider-metal3-k8s/crds/apiextensions.k8s.io_v1_customresourcedefinition_metal3clusters.infrastructure.cluster.x-k8s.io.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       clientConfig:
         service:
           name: capm3-webhook-service
-          namespace: multicluster-engine
+          namespace: '{{ .Release.Namespace }}'
```

Comment:
This is due the
`namespace: '{{ .Release.Namespace }}'` stanza in the kustomization.yaml
Although it generates with `{{ .Release.Namespace }}` actual value in the midstream template (within backplane operator), the final render has the desired output.